<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Keyb_E">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_A">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_B">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_F">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_C">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_D">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_G">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_Cs">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>C#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_Ds">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>D#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_Fs">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>F#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_Gs">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>G#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Keyb_As">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>A#</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
